### PR TITLE
feat(test): change test pkg name and codecov

### PR DESCRIPTION
**Pull request by hunshenshi**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -38,11 +38,11 @@ jobs:
 
       - name: Run Unit tests.
         # skip integration testing
```
Comment (huangzhiran) [inline .github/workflows/check.yml]: skip e2e test

File: .github/workflows/check.yml [hunk 2]
```diff
@@ -38,11 +38,11 @@ jobs:
...
-        run: go test -p 1 -gcflags="all=-N -l" `go list ./... | grep -v github.com/machinefi/sprout/cmd/test` -covermode=atomic -coverprofile cover.out
+        run: go test -p 1 -gcflags="all=-N -l" `go list ./... | grep -v github.com/machinefi/sprout/cmd/e2etest` -covermode=atomic -coverprofile cover.out
```
Comment (huangzhiran) [inline .github/workflows/check.yml]: can del "-p 1"

File: cmd/e2etest/e2e_test.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package test
+package e2etest
```
Comment (huangzhiran) [inline cmd/e2etest/e2e_test.go:1]: maybe we can use e2e_test name. for go pkg name is "wordword", but cmd can be "word_word"

File: .github/workflows/check.yml (hunk 2 above)
Comment (hunshenshi) [reply .github/workflows/check.yml]: gomonkey is not thread safe, so can not del `-p 1`

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**